requirements.txt

```diff
@@ -21,7 +21,7 @@ django-slack==5.19.0
 git+https://github.com/DefectDojo/django-tagging@develop#egg=django-tagging
 django-watson==1.6.3
 django-prometheus==2.3.1
-Django==5.0.8
+Django==5.1.1
```

Comment:
There shouldn't be  an update to Django in this parser change - please remove.

Reply:
Thank you for the catch - I am assuming this was something from a different commit when I did a rebase against dev.  It should only be the 3 files - nothing in this one.  I've rebased it against the latest dev branch.

Reply:
No worries